modules/pom.xml

```diff
@@ -247,11 +247,15 @@
                     <include>api/indices.analyze.json</include>
                     <include>api/indices.create.json</include>
                     <include>api/indices.refresh.json</include>
+                    <include>api/indices.put_mapping.json</include>
```

Comment:
These weird incantations are required to allow plugins to use certain APIs in REST tests. This wasn't something that we particularly liked so we fixed it in master a long time ago.